service/src/main/java/uk/gov/hmcts/reform/fpl/service/OthersService.java

```diff
@@ -129,4 +125,12 @@ private Other getFirstOther(List<Element<Other>> confidentialOthers, List<Elemen
         }
         return null;
     }
+
+    // This removes firstOther element in others.
```

Comment:
Method does more than the comment implies. Fairly simple method, worth removing.

Reply:
hmm, the comment is there just to explain than a little more than a simple get is going on. I will leave as is